src/modules/sensors/module.yaml

```diff
@@ -0,0 +1,795 @@
+          14: Roll 180°, Yaw 270°
+          15: Roll 180°, Yaw 315°
+          16: Roll 90°
+          17: Roll 90°, Yaw 45°
+          18: Roll 90°, Yaw 90°
+          19: Roll 90°, Yaw 135°
+          20: Roll 270°
+          21: Roll 270°, Yaw 45°
+          22: Roll 270°, Yaw 90°
+          23: Roll 270°, Yaw 135°
+          24: Pitch 90°
+          25: Pitch 270°
+          26: Pitch 180°, Yaw 90°
+          27: Pitch 180°, Yaw 270°
+          28: Roll 90°, Pitch 90°
+          29: Roll 180°, Pitch 90°
+          30: Roll 270°, Pitch 90°
+          31: Roll 90°, Pitch 180°
+          32: Roll 270°, Pitch 180°
+          33: Roll 90°, Pitch 270°
+          34: Roll 180°, Pitch 270°
+          35: Roll 270°, Pitch 270°
+          36: Roll 90°, Pitch 180°, Yaw 90°
+          37: Roll 90°, Yaw 270°
+          38: Roll 90°, Pitch 68°, Yaw 293°
+          39: Pitch 315°
+          40: Roll 90°, Pitch 315°
+        min: -1
+        max: 40
+        reboot_required: true
+        category: System
+        default: -1
+      CAL_MAG0_XOFF:
+        description:
+          short: Magnetometer X-axis offset
+          long: Magnetometer X-axis offset
+        category: system
+        type: float
+        default: 0.0
+        volatile: true
+      CAL_MAG0_YOFF:
+        description:
+          short: Magnetometer Y-axis offset
+          long: Magnetometer Y-axis offset
+        category: system
+        type: float
+        default: 0.0
+        volatile: true
+      CAL_MAG0_ZOFF:
+        description:
+          short: Magnetometer Z-axis offset
+          long: Magnetometer Z-axis offset
+        category: system
+        type: float
+        default: 0.0
+        volatile: true
+      CAL_MAG0_XSCALE:
+        description:
+          short: Magnetometer X-axis scaling factor
+          long: Magnetometer X-axis scaling factor
+        category: system
+        type: float
+        default: 1.0
+        volatile: true
+      CAL_MAG0_YSCALE:
+        description:
+          short: Magnetometer Y-axis scaling factor
+          long: Magnetometer Y-axis scaling factor
+        category: system
+        type: float
+        default: 1.0
+        volatile: true
+      CAL_MAG0_ZSCALE:
+        description:
+          short: Magnetometer Z-axis scaling factor
+          long: Magnetometer Z-axis scaling factor
+        category: system
+        type: float
+        default: 1.0
+        volatile: true
+      CAL_MAG0_XODIAG:
+        description:
+          short: Magnetometer X-axis off diagonal factor
+          long: Magnetometer X-axis off diagonal factor
+        category: system
+        type: float
+        default: 0.0
+        volatile: true
+      CAL_MAG0_YODIAG:
+        description:
+          short: Magnetometer Y-axis off diagonal factor
+          long: Magnetometer Y-axis off diagonal factor
+        category: system
+        type: float
+        default: 0.0
+        volatile: true
+      CAL_MAG0_ZODIAG:
```

Comment:
multi?

Reply:
Oh whoops completely forgot about using multi there :smile: I'll fix that